MOBILE_API.xml

```diff
@@ -2872,6 +2872,15 @@
         <param name="hapticSpatialDataSupported" type="Boolean" mandatory="false">
             <description>True if the system can utilize the haptic spatial data from the source being streamed. If not included, it can be assumed the module doesn't support haptic spatial data'. </description>
         </param>
+        <param name="diagonalScreenSize" type="Float" mandatory="false" since="6.0">
```

Comment:
Need `minvalue` of `1` per [proposal revisions](https://github.com/smartdevicelink/sdl_evolution/issues/808)